config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -14,7 +14,8 @@ presubmits:
     run_if_changed: 'staging/src/k8s.io/apiserver/pkg/storage/value/|staging/src/k8s.io/kms/|staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/|test/e2e/testing-manifests/auth/encrypt/'
     path_alias: k8s.io/kubernetes
     branches:
-    - ^master$ # TODO(aramase): enable for release branches
+    - ^master$
+    - release-1.27
```

Comment:
Can we use a regex to make this match 1.27 and greater?

Reply:
updated it! the regex will handle until `release-1.99`. We can update it after that 😉

Reply:
added regex for until `release-1.99`. We'll need to update this for 2.0 😄